tool/refresh-code-excerpts.sh

```diff
@@ -30,8 +30,8 @@ FRAG="$TMP/_fragments"
 if [[ -e "$FRAG" ]]; then echo Deleting old "$FRAG"; rm -Rf "$FRAG"; fi
 
 ARGS+='--yaml '
-if [[ ! -e "pubspec.lock" ]]; then pub get; fi
-pub run build_runner build --delete-conflicting-outputs --config excerpt --output="$FRAG"
+if [[ ! -e "pubspec.lock" ]]; then dart pub get; fi
```

Comment:
🤷  unrelated code change ☝️

Reply:
Yep, it is... I keep forgetting to do it in a separate PR, and it keeps biting me when I do local testing (since I don't have a separate Dart SDK downloaded).

Reply:
no worries. I do it all the time. I just have a little @natebosch sitting on my shoulder now...when I review *other peoples* code 😁